db_store.py

```
@@ -436,14 +437,18 @@ def defrag_availability(db_conn=None):
                         c.execute(
-                            f"UPDATE {table} SET end_time = ? WHERE id = ?",
+                            f"UPDATE {table} SET end_time = ? WHERE id = ?",  # nosec B608
                             (new_end, prev_row_id),
                         )
                         prev_end = new_end
 
                     # Delete current block
-                    c.execute(f"DELETE FROM {table} WHERE id = ?", (curr_row_id,))
+                    # Table name is an internal constant, safe from injection
+                    c.execute(
+                        f"DELETE FROM {table} WHERE id = ?", (curr_row_id,)
```

Ruff (0.15.0)

[error] Possible SQL injection vector through string-based query construction

(S608)